src/net/sourceforge/kolmafia/textui/langserver/AshLanguageServer.java

```diff
@@ -61,6 +66,9 @@ public static AshLanguageServer launch(final InputStream in, final OutputStream
       new StateCheckWrappers.AshWorkspaceService(this);
 
   public final ExecutorService executor = Executors.newCachedThreadPool();
+  public final FilesMonitor monitor = new FilesMonitor(this);
+
+  public final Map<File, Script> scripts = Collections.synchronizedMap(new Hashtable<>(20));
```

Comment:
Why Hashtable? Might be the right move, but when HashMap is the standard anything else will look odd.

Reply:
HashMap permits null values, which we don't want here. Otherwise it could have been a HashMap (given that we're doing the synchronization ourselves)

Reply:
Could you please add a comment to the code with that reasoning?